build.gradle.kts

```diff
@@ -0,0 +1,185 @@
+import se.bjurr.gitchangelog.plugin.gradle.GitChangelogTask
+import kotlin.io.println
+
+plugins {
+  id("java-gradle-plugin")
+  id("groovy")
+  id("maven-publish")
+  id("signing")
+  id("net.researchgate.release") version "2.8.1"
+  id("se.bjurr.gitchangelog.git-changelog-gradle-plugin") version "1.64"
+}
+
+gradlePlugin {
+  plugins {
+    register("gradleDocker") {
+      id = "com.ekino.oss.gradle.plugin.docker"
+      implementationClass = "com.ekino.oss.gradle.plugin.docker.DockerPlugin"
+    }
+  }
+}
+
+repositories {
+  mavenLocal()
+  jcenter()
+  gradlePluginPortal()
+}
+
+group = "com.ekino.oss.gradle.plugin"
+description = "Gradle Plugin for Docker management"
+version = "0.0.1-SNAPSHOT"
+
+val dockerComposePluginVersion = "0.9.4"
+val junitVersion = "5.5.1"
+val ossrhUsername = ""
+val ossrhPassword = ""
+val ossrhUrl = ""
```

Comment:
I don't think it's going to work this way. You should have something like that :
```kotlin
repositories {
    maven {
      val ossrhUrl: String? by project
      val ossrhUsername: String? by project
      val ossrhPassword: String? by project

      url = uri(ossrhUrl ?: "")

      credentials {
        username = ossrhUsername
        password = ossrhPassword
      }
    }
  }
```

Reply:
fixed